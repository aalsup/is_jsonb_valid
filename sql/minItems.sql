-- minItems validation
-- longer is valid
SELECT is_jsonb_valid('{"minItems":1}', '[1,2]');
-- exact length is valid
SELECT is_jsonb_valid('{"minItems":1}', '[1]');
-- too short is invalid
SELECT is_jsonb_valid('{"minItems":1}', '[]');
-- ignores non-arrays
SELECT is_jsonb_valid('{"minItems":1}', '""');