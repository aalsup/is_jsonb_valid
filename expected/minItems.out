-- minItems validation
-- longer is valid
SELECT is_jsonb_valid('{"minItems":1}', '[1,2]');
 is_jsonb_valid 
----------------
 f
(1 row)

-- exact length is valid
SELECT is_jsonb_valid('{"minItems":1}', '[1]');
 is_jsonb_valid 
----------------
 f
(1 row)

-- too short is invalid
SELECT is_jsonb_valid('{"minItems":1}', '[]');
 is_jsonb_valid 
----------------
 f
(1 row)

-- ignores non-arrays
SELECT is_jsonb_valid('{"minItems":1}', '""');
 is_jsonb_valid 
----------------
 f
(1 row)
